lib/ex_unit/lib/ex_unit/server.ex

```diff
@@ -21,11 +20,13 @@ defmodule ExUnit.Server do
   end
 
   def take_async_cases(count) do
-    GenServer.call(__MODULE__, {:take_async_cases, count}, @timeout)
+    timeout = Application.get_env(:ex_unit, :timeout)
```

Comment:
What happens when this key has not been set in the application's environment? I think you might want the three-argument version of `Application.get_env` here.